### ISSUE TITLE
fuzzy file search

### ISSUE THREAD

**Issue by justpeanuts** [MM2-0/Kvaesitso#145]
there is no need for a completely fuzzy implemetation (like the albert launcher for linux does) rather just the fuzzy matching that rofi (a simple dmenu like launcher for linux) does by default would be very useful. Use case : i) one usually only remembers certain keywords in the file and one doesn't usually remembers if the keywords appear consequtively in the filename. ii) special characters like underscore or hyphen could be used instead of white-spaces in some files.

**Comment by MM2-0** [comment]
I agree that this would be useful. Unfortunately, the local file search relies on the Android MediaStore API which does not provide a possibility to perform fuzzy searches in an efficient way. The same limitation applies to the contacts and calendar search.
